### PR TITLE
KAZOO-4470: Run start_dtmf in case of "101 telephone-event" absence

### PR DESCRIPTION
**Pull request by onnet**
<<"One gateway that is connecting to Kazoo is using inband DTMF (changing to RFC2833 or INFO is not possible for them). On a standard freeswitch server, this is solved by using <action application="start_dtmf"> in the extension . Is the equivalent possible with Kazoo?">> 

Please check 2600-dev for details 

https://groups.google.com/forum/#!searchin/2600hz-dev/kiril/2600hz-dev/jV7UWPnMyKc/2qbM4nKmAwAJ

### CODE REVIEW

File: applications/callflow/src/cf_route_req.erl [hunk 1]
```diff
@@ -236,14 +236,22 @@ get_ringback_media(Flow, JObj) ->
...
         andalso whapps_call:authorizing_type(Call) =:= 'undefined'
     of
         'false' -> <<"none">>;
-        'true' -> <<"ring_ready">>
+        'true' -> check_dtmf_type(JObj) 
+    end.
+
+-spec check_dtmf_type(wh_json:object()) -> ne_binary().
+check_dtmf_type(JObj) ->
+    SDPRemote = wh_json:get_value(<<"Remote-SDP">>, JObj),
```
Comment (fenollp) [inline applications/callflow/src/cf_route_req.erl]: No default value?

File: applications/callflow/src/cf_route_req.erl [hunk 2]
```diff
@@ -236,14 +236,22 @@ get_ringback_media(Flow, JObj) ->
...
+    case re:run(SDPRemote, <<"101 telephone-event">>, []) of
```
Comment (fenollp) [inline applications/callflow/src/cf_route_req.erl]: Use `binary:match(es)`

**Comment by jamesaimonetti** [comment]
@onnet I think moving the SDP check to ecallmgr_fs_route (once the `route_resp` is received) would work better. Then trunkstore and other apps get it for free!

It should also be behind an `ecallmgr_config:is_true(<<"should_detect_inband_dtmf">>)` or similar, so people can opt in to the addition. 

Thoughts?

**Comment by onnet** [comment]
@jamesaimonetti Thanks for a hint. This way it collapsed to a couple of lines. Will close/reopen this PR to make it more transparent.

**Pull request closed without merging**